src/client.rs

```diff
@@ -568,4 +568,44 @@ impl ChannelsAPI for AriClient {
         eval_status_code!(status, StatusCode::CREATED, Some(body_str));
         Ok(())
     }
+
+    async fn raise_dtmf(
```

Comment:
still didn't have chance to test previous change. please for now rollback this commit. should be addressed separately. i will try to test recording this week.